src/modules/commander/Commander.hpp

```diff
@@ -192,6 +213,12 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 	Subscription<mission_result_s>			_mission_result_sub{ORB_ID(mission_result)};
 	Subscription<vehicle_global_position_s>		_global_position_sub{ORB_ID(vehicle_global_position)};
 	Subscription<vehicle_local_position_s>		_local_position_sub{ORB_ID(vehicle_local_position)};
+	Subscription<sensor_bias_s>			_sensor_bias_sub{ORB_ID(sensor_bias)};
+	Subscription<airspeed_s>			_airspeed_sub{ORB_ID(airspeed)};
+
+	struct airspeed_s _airspeed = {};
```

Comment:
The Subscription wrappers above contain both a copy of the last data update and the subscription handling. You can get rid of _airspeed and _sensor_bias.